.github/workflows/rs_ci.yml

```diff
@@ -139,7 +139,7 @@ jobs:
           tool: nextest@0.9
       - uses: mozilla-actions/sccache-action@v0.0.9
       - working-directory: bottlecap
-        run: cargo nextest run --workspace
+        run: cargo nextest run --workspace --features integration_test
```

Comment:
Why was this changed?

Reply:
https://github.com/DataDog/datadog-lambda-extension/pull/684#discussion_r2121704198